### ISSUE TITLE
Directory App: Using search text shows no collections in case a matching biobank is found

### ISSUE THREAD

**Issue by dtroelofsprins** [molgenis/molgenis-emx2#3082]
**What version of EMX2 are you using (see footer)**
v10.5.1

**Describe the bug**
Directory App: When using text search on biobank f.e. Wroclaw, the biobank is found, it only says "This biobank has no collections yet." which is not true. It has 9 collections. See: https://directory.bbmri-eric.eu/#/catalogue?search=wroclaw

**To Reproduce**
1. Go to 'https://bbmri-emx2-test.molgeniscloud.org/Directory/directory/#/catalogue?search=Wroclaw
2. See error

**Expected behavior**
I expect to see the collections that belong to this biobank: See: https://directory.bbmri-eric.eu/#/catalogue?search=wroclaw

**Comment by MaxPostema** [comment]
I think this is working as intended... Just not what is expected...  if you search for "Phase 1" you will find "Amsterdam UMC Biobank: Location AMC" with only one collection "A Phase 1 Clinical Trial" All the other 70+ collections are hidden.

If you search for 'wroclaw' you will find the biobank 'wroclaw' that contain no 'wroclaw' in the collections.

I think, while searching the text '1 collection available' needs to be '1 collection found'  and 'This biobank has no collections yet.' needs to be 'no collections found.' To reflect that the collections are also searched.

We can also say that if there are no collections found, every collection needs to be displayed.

TL;DR. To fix this issue we will need to define the correct behavior. It is now searching in the biobanks and collections as intended.

**Comment by MaxPostema** [comment]
We decided to first only change the texts. And after that see if more needs to be done.
